docs/source/guides/network_node/ursula_configuration_guide.rst

```diff
@@ -17,6 +17,40 @@ The worker's ethereum account must remain unlocked to send automated work confir
 ether to pay for transaction gas; however, it is *not* necessary (and potentially risky) to hold NU tokens on a worker's
 account for any reason.
 
+
+Worker Requirements
+-------------------
+
+Aside from the :ref:`base requirements <base-requirements>` for installation of the ``nucypher`` library:
+
+* 4GB RAM (minimum)
+* 2 CPUs (minimum) - x86 architecture
+* HDD storage requirement grows with network size - information on each node requires ~2KB of storage
+* Static public IP address - NAT management where applicable
```

Comment:
Is `static` fair, since it is allowed for the IP to change, but then the worker's configuration would need to be updated?

Reply:
Hmmm, certainly much more desirable to have a static IP, but its not *technically* required.

Reply:
✔️